pyproject.toml

```diff
@@ -113,6 +114,7 @@ include = ["flopy", "flopy.*"]
 
 [tool.setuptools.package-data]
 "flopy.mf6.data" = ["dfn/*.dfn"]
+"flopy.mf6.utils.codegen" = ["templates/*.jinja"]
```

Comment:
package-data is used to send files into the package on install right? Since the generate-classes part is still developer only, I would say we don't need jinja files in the final deployment.

Reply:
I think since it's available to all flopy users now it should probably stay that way. It is typically a developer task but we have used it when teaching classes too, for example, where people may have the pypi package not a source install